certified-connectors/Projectplace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Projectplace",
+    "description": "Get work done across projects and teams\nProjectplace lets you plan, collaborate and track progress of all projects and assignments in an easy and user-friendly way.",
```

Comment:
Could this all be one line? So `Get work done across projects and teams. Projectplace lets you plan, collaborate...` That would help with readability in certain parts of the platform.

Reply:
Changed to one line